.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Stale issues Bot'
+
+on:
+  schedule:
+    - cron: '0 8 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    
+    permissions:
+      issues: write
+    
+    steps:
+      - uses: actions/stale@v6
+        with:
+          exempt-issue-labels: 'do-not-stale'
+          stale-issue-message: 'This issue has been automatically marked as stale because it has been without activity for 90 days. This issue will be closed in 14 days unless you comment or remove the stale label.'
+          close-issue-message: 'This issue was closed because it was stale for 14 days without any activity.'
```

Comment:
```suggestion
          close-issue-message: 'This issue was closed because it was stale for 14 days without any activity. Please re-open the issue if you feel it has not been resolved.'
```